.travis.yml

```diff
@@ -10,8 +13,10 @@ env:
   - TEST_SUITE=./spec/build_spec.cr
   - TEST_SUITE=./spec/amber
 
-script: 
-  - docker-compose run --service-ports spec bash -c "cd /opt/amber && crystal spec $TEST_SUITE -D run_build_tests"
+script:
+  - crystal spec
+  - bin/ameba
+  - docker-compose run --service-ports spec bash -c "cd /app/user && crystal spec $TEST_SUITE -D run_build_tests"
```

Comment:
change this to:
```
 - docker-compose run bash -c "crystal spec $TEST_SUITE -D run_build_tests"
```

@eliasjpr @elorest do either of you know why we added the `--service-ports spec`?  I don't see any ports defined in `docker-compose.yml` for spec so not sure what this flag is for.  If not, we can remove this setting.